apps/communications/dialer/index.html

```diff
@@ -36,6 +36,7 @@
     <link rel="stylesheet" type="text/css" href="/shared/elements/gaia-icons/gaia-icons.css">
 
     <link rel="stylesheet" type="text/css" href="/shared/style/option_menu.css">
+    <link rel="stylesheet" href="/shared/pages/import/style/curtain_frame.css">
```

Comment:
It doesn't. This is because the curtain refactor in Contacts (and it belongs to Contacts). Once Contacts will be out of `Communications` we must remove this line. I'll add a line explaining the scenario.